Microsoft.DotNet.Interactive.Rendering/TerminalFormatter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Interactive.Rendering
+{
+    public class TerminalFormatter
+    {
+        public static ITypeFormatter Create(
+            Type type,
+            bool includeInternals = false)
+        {
+            var genericCreateForAllMembers = typeof(TerminalFormatter<>)
+                .MakeGenericType(type)
+                .GetMethod(nameof(TerminalFormatter<object>.Create), new[]
+                {
+                    typeof(bool)
+                });
+
+            return (ITypeFormatter)genericCreateForAllMembers.Invoke(null, new object[]
+            {
+                includeInternals
+            });
+        }
+
+        public const string MimeType = "text/vt100";
```

Comment:
This is probably not the correct name for this mime type, since VT-100 is really a subset of the [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code) that might be used.